client/src/slice.ts

```diff
@@ -1,36 +1,33 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-import { helloQuery } from './queries/hello';
+import { loginDriver } from './queries/login';
 
 const { actions, reducer } = createSlice({
   name: 'app',
   initialState: {
-    loginIdInput: '',
-    loginPasswordInput: '',
-    helloString: '',
+    loginField: {
+      email: '',
+      password: '',
+    },
   },
   reducers: {
-    setLoginIdInput(state, { payload }) {
-      return payload;
-    },
-    setLoginPasswordInput(state, { payload }) {
-      return payload;
+    setLoginEmail(state, { payload: email }) {
+      return { ...state, loginField: { ...state.loginField, email } };
     },
-    setHelloString(state, { payload }) {
-      return payload;
+    setLoginPassword(state, { payload: password }) {
+      return { ...state, loginField: { ...state.loginField, password } };
     },
   },
 });
 
-export const getHello = (client: any) => async (dispatch: any) => {
+export const requestLogin = (client: any) => async (dispatch: any, getState : any) => {
+  const { loginField } = getState();
   const { data } = await client.query({
-    query: helloQuery,
+    query: loginDriver(loginField.email, loginField.password),
```

Comment:
이 부분 `mutation` 으로 변경해야 할 것 같습니당
```suggestion
const { data } = await client.mutate({
    mutation: loginDriver(loginField.email, loginField.password),
```